log/log.go

```diff
@@ -41,6 +42,26 @@ func NewLogger(logLevel Level) *Logger {
 	return NewLogrusLogger(logrusLog)
 }
 
+// NewCronLogger creates an instance of *Logger.
+func NewCronLogger(logLevel Level) *Logger {
+	logrusLog := logrus.New()
+
+	// Configure logger
+	// Don't log to stderr by default (stackdriver treats stderr as error severity)
+	logrusLog.SetOutput(os.Stdout)
```

Comment:
Is this still true now that you specify the severity field below?
https://stackoverflow.com/a/44187839

Reply:
Makes sense to me to remove.